store/store.go

```diff
@@ -424,7 +424,7 @@ func (s Store) RenderTreeStore(key string, rebuild bool) error {
 	if err := s.treestore.Remove(key); err != nil {
 		return err
 	}
-	if err := s.treestore.Write(key, &s); err != nil {
+	if err := s.treestore.Write(key, &s, 0, 0); err != nil {
```

Comment:
I don't think we ever need to write to the treestore with a uid_shift so these parameters are not needed.